src/model/file.py

```diff
@@ -57,7 +57,7 @@ class File:
 
     def __init__(self, name: str, spdx_id: str, checksums: List[Checksum], file_type: List[FileType] = None,
                  concluded_license: Optional[Union[LicenseExpression, SpdxNoAssertion, SpdxNone]] = None,
-                 license_info_in_file: List[Union[LicenseExpression, SpdxNoAssertion, SpdxNone]] = None,
+                 license_info_in_file: Optional[Union[List[LicenseExpression], SpdxNoAssertion, SpdxNone]] = None,
```

Comment:
Good point, but I think we should have `NOASSERTION` as default value for all optional licenses as stated [here](https://spdx.github.io/spdx-spec/v2.3-RC1/diffs-from-previous-editions/):
> Update Clause 7, 8, and 9 to make several of the licensing properties optional rather than requiring the use of "NOASSERTION" when no value is provided.